docs/governance/community-guidelines/ways-of-working.md

```diff
@@ -0,0 +1,32 @@
+# Ways of Working
+
+This document is a [community guideline].
+
+The document outlines the general workflow and describes expected interactions between various working groups ([WG]s).
+
+**Community-Driven Direction:**
+   - The project's direction will be influenced by community needs.
+   - Requests will be processed by the [Communications WG].
+   - The [Communications WG] will collaborate with the [SC] to establish the project's high-level roadmap.
+
+**Service Cataloging:**
+   - The [Communications WG] will communicate the roadmap to the [Taxonomy WG].
+   - Based on the roadmap, the [Taxonomy WG] will expand the service catalog.
+   - Completed services will be handed over to the [Security WG].
+
+**Security Cataloging:**
+   - The [Security WG] will catalog security threats and controls for each service.
```

Comment:
I think the section demarkation has become somewhat unclear for me at this point. What are we trying to do with the three service/security/release subsections here?

Reply:
For me I see there as 4 main stages of the flow, requests come in, services are catalogged, security threats/controls are catalogged, updates are released so I was just putting it down in that way. Happy to adjust to something better.

Reply:
Maybe just adding a clearer sequential relationship will communicate it better?

Also,  project drivers may come from end user requests or contributor motivation... Not sure we need to capture that, but it might color how we phrase things.

Still noodling this 🤔 We'll need to compare it with the project charters for factual accuracy and language precision. Don't take my commentary below as change requests/suggestions.

1. The Taxonomy WG creates a list of common cloud services and their common features. Other WG feedback helps guide the format and structure of the project taxonomy. Contributor motivations and end user requests guide the prioritization of service additions.
2. The Security WG extends the taxonomy to include a list of threats and controls for each service. Other WG feedback helps guide the format and structure of the threats and controls documentation. Contributor motivations and end user requests guide the prioritization of services to create threats and controls for.
3. The Delivery WG takes feedback from other WGs to determine the best process for releasing a version controlled taxonomy, and maintains any related automation elements.
4. The Communications WG takes feedback and release plans from other WGs to deliver a public project roadmap and any relevant public communications. Also, the Comms WG relays any feedback or requests from end users to the correct WG.

Reply:
In my notes I had a diagram of the flow which might be more what you are thinking. I will try and put it into an actual diagram and see what that looks like.

I think I want to try to avoid having too many words here as it would just be another place to keep up to date.

Reply:
I have pushed another version, I tried playing around with diagrams for about a day but couldn't quite get something that felt right. This version just has a reworded, simplified(?) version of the basic workflow and just includes the WGs involved in the critical path.  Would be keen to hear your thoughts @eddie-knight. I have been back and forward on this so many times, going round in circles, just can't quite nail it.

Reply:
I like what you have here, but I'm starting to wonder whether we should hand this off to the Comms WG to maintain- maybe it doesn't need to be a guideline at all?

@Alexstpierrework and @sshiells-scottlogic does it make sense for you guys to collab next week, to see how we can consolidate all the different threads related to "what is the new structure?"

Reply:
Yeah, @Alexstpierrework  if you have some time today, Tues or Wed that would be great. I am out on vacation Thursday for the reaminder of this week and the following week.

Reply:
@sshiells-scottlogic I have availability today between 3-5GMT, if this is too quick of a turnaround, I can give some times for Wednesday.

Reply:
Just to confirm do you mean GMT as actual GMT, or UK time (which is currently GMT+1).

We have the steer co meeting at 2pm UK (BST / GMT + 1) - I am clear after that.

Reply:
Just trying to find a baseline.

UK time 4:00 works. I will send a zoom link.